### ISSUE TITLE
Global Navigation Bar colors get applied in the DropIn

### ISSUE THREAD

**Issue by FranciPC7** [braintree/braintree-ios-drop-in#435]
### Braintree Drop-in SDK Version

9.10.0

### Environment

Sandbox

### Xcode Version

Xcode 15

### OS Version & Device

iPhone 12 - iOS 17

### Integration type

CocoaPods

### Development Processor

Apple Silicon (M-series chips)

### Describe the bug

In SwiftUI it is common practice to define global navigation bar colors via AppDelegate:

```swift
let navBarAppearance = UINavigationBarAppearance()
        let backItemAppearance = UIBarButtonItemAppearance()
        backItemAppearance.normal.titleTextAttributes = [.foregroundColor : UIColor.colorToolsText] // fix text color
        navBarAppearance.backButtonAppearance = backItemAppearance
        let image = UIImage(systemName: "chevron.backward")?.withTintColor(.colorToolsText, renderingMode: .alwaysOriginal) // fix indicator color
        navBarAppearance.setBackIndicatorImage(image, transitionMaskImage: image)
        
        navBarAppearance.configureWithOpaqueBackground()
        navBarAppearance.titleTextAttributes = [.foregroundColor: UIColor.colorToolsText]
        navBarAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor.colorToolsText]
        navBarAppearance.backgroundColor = .colorPrimaryDark
        UINavigationBar.appearance().standardAppearance = navBarAppearance
        UINavigationBar.appearance().scrollEdgeAppearance = navBarAppearance
        
        UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).backgroundColor = .white.withAlphaComponent(0.15)
``` 

In the DropIn version 9.* the DropIn takes over the default color which results in unconsistent color scheming and we can't get back to usage of default navigation bar colors.

![IMG_1942](https://github.com/braintree/braintree-ios-drop-in/assets/94113952/00878bf3-c3e5-4fec-9668-f573a8e110ed)

### To reproduce

1. Have Global navigation bar colors defined
AppDelegate.swift
```swift
let navBarAppearance = UINavigationBarAppearance()
        let backItemAppearance = UIBarButtonItemAppearance()
        backItemAppearance.normal.titleTextAttributes = [.foregroundColor : UIColor.colorToolsText] // fix text color
        navBarAppearance.backButtonAppearance = backItemAppearance
        let image = UIImage(systemName: "chevron.backward")?.withTintColor(.colorToolsText, renderingMode: .alwaysOriginal) // fix indicator color
        navBarAppearance.setBackIndicatorImage(image, transitionMaskImage: image)
        
        navBarAppearance.configureWithOpaqueBackground()
        navBarAppearance.titleTextAttributes = [.foregroundColor: UIColor.colorToolsText]
        navBarAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor.colorToolsText]
        navBarAppearance.backgroundColor = .colorPrimaryDark
        UINavigationBar.appearance().standardAppearance = navBarAppearance
        UINavigationBar.appearance().scrollEdgeAppearance = navBarAppearance
        
        UITextField.appearance(whenContainedInInstancesOf: [UISearchBar.self]).backgroundColor = .white.withAlphaComponent(0.15)
``` 

2. Initiate Drop In controller and the colors will follow te global design, no way to use the default colors. Also the cancel button color can't be set.

### Expected behavior

I would expect to have option to keep the original design of the DropIn regardless the global config or atleast have the option to set more color options of the navigation bar of the controller.

### Screenshots

![IMG_1942](https://github.com/braintree/braintree-ios-drop-in/assets/94113952/26e502b1-be49-4d0b-bb57-389062dfe331)

**Comment by jaxdesmarais** [comment]
Hello @FranciPC7 -

Thanks for bringing this to our attention, I can reproduce the same behavior in a `UIKit` view as well. This appears to be an oversight on our end as other navigation bar items within the SDK are customizable via `BTDropInUICustomization`. We will work on getting this bug addressed so you can pass in customizations or a default via `BTDropInUICustomization`.

**Comment by jaxdesmarais** [comment]
Hey @FranciPC7 -

The fix for this has been released in [Drop-in v9.10.1](https://github.com/braintree/braintree-ios-drop-in/releases/tag/9.10.1). Additionally, if desired you can use `BTDropInRequest.uiCustomization` to customize the color scheme, otherwise the default will be used. Thanks again for bringing this bug to our attention and using the SDK!
